frontend/src/pages/Admin/Components/Faq/ManageFaq/ManageFaq.jsx

```diff
@@ -1,86 +1,159 @@
-import React from "react"
+import React from "react";
 import { useEffect, useState } from "react";
 import Accordion from "@material-ui/core/Accordion";
 import AccordionDetails from "@material-ui/core/AccordionDetails";
 import AccordionSummary from "@material-ui/core/AccordionSummary";
 import Typography from "@material-ui/core/Typography";
-import Button from "@material-ui/core/Button"
-import { Edit, Delete } from "@material-ui/icons"
+import Button from "@material-ui/core/Button";
+import { Edit, Delete } from "@material-ui/icons";
 import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
 import { END_POINT } from "../../../../../config/api";
 import Loader from "../../../../../components/util/Loader";
-import style from "./manage-faq.module.scss"
-export function ManageFaq() {
-    const [faqs, setFaqs] = useState([]);
-    const [expanded, setExpanded] = React.useState(false);
-    const [isFetching, setIsFetching] = useState(false)
-    const handleChange = (panel) => (event, isExpanded) => {
-        setExpanded(isExpanded ? panel : false);
+import style from "./manage-faq.module.scss";
+import { SimpleToast } from "../../../../../components/util/Toast";
+
+export function ManageFaq(props) {
+  const [faqs, setFaqs] = useState([]);
+  const [expanded, setExpanded] = useState(false);
+  const [isFetching, setIsFetching] = useState(true);
+  const [open, setOpenToast] = useState(false);
+  const [toastMessage, setToastMessage] = useState("");
+  const [severity, setSeverity] = useState("success");
+  const [reload, setReload] = useState(true);
+  const handleCloseToast = () => {
+    setTimeout(() => {
+      setOpenToast(false);
+    }, 500);
+  };
+  const handleChange = (panel) => (event, isExpanded) => {
+    setExpanded(isExpanded ? panel : false);
+  };
+  async function fetchAllFaq() {
+    try {
+      const response = await fetch(`${END_POINT}/getFaq`);
+      const data = await response.json();
+      console.log(data);
+      setFaqs(data.Faq);
+      setIsFetching(false);
+    } catch (err) {
+      console.log(err.message);
+      setIsFetching(false);
+    }
+  }
+
+  const deleteFaq = async (faqId) => {
+    setIsFetching(true);
+    const url = `${END_POINT}/deleteFaq`;
+    const body = {
+      userId: props.adminData._id,
```

Comment:
delete api expecting adminId in body is wrong, @SwayamRana808 can you fixx https://github.com/HITK-TECH-Community/Community-Website/blob/main/backend/app/routes/faq/deleteFaq.js api using https://github.com/HITK-TECH-Community/Community-Website/pull/933 as reference, you can create a separate issue for the same. Till then we will keep this pr on hold

Reply:
So i should remove adminId and add authmiddleware right? [issue](https://github.com/HITK-TECH-Community/Community-Website/issues/952)

Reply:
yes correct @SwayamRana808